_templates/component/new/sass-styles.ejs.t

```diff
@@ -0,0 +1,5 @@
+---
+to: src/components/<%= h.inflection.dasherize(component_name) %>/_index.scss
+---
+
+.c-<%= h.inflection.dasherize(component_name) %> {}
```

Comment:
Can we note to add in the needed Sass library function/mixin here - maybe create a sub task on the current ticket to update once the Sass library ticket has been completed

Reply:
will do

Reply:
https://arcpublishing.atlassian.net/browse/TMEDIA-650